dist/Model.js

```diff
@@ -0,0 +1,7 @@
+class Model {
+    getPlayers(data) {
+        return $.get(`/teams/${data}`, function (players) {
```

Comment:
If you are using promises (in the `mian.js`) then you don't need the callback function here. It's either callbacks **or** promises (not both).